docs/UserGuide.md

```diff
@@ -349,54 +377,139 @@ Examples:
 
+**Format:**
 ```
-deletestu INDEX
+yes
 ```
 
-* Deletes the student at the specified `INDEX`.
+**Expected result:**
+```
+Deleted student: PARTICULARS_OF_STUDENT_AT_SPECIFIED_INDEX
+```
+   ![ScreenshotDeletestu](images/ScreenshotDeletestu.png)
+
+Examples:
+* `liststu` followed by `deletestu 2` followed by `yes` deletes the 2nd student in the student list.
+* `findstu Betsy` followed by `deletestu 1` followed by `y` deletes the 1st student in the results of the `findstu Betsy` command.
+
+{ end of `features#student#deletestu` written by: Marcus Tan Wei }
+
+{ start of `features#student#editstu` written by: Marcus Tan Wei }
+
+#### 5.3.5. Editing a student's particulars : `editstu`
+
+<div markdown="block" class="alert alert-info">
+
+**:information_source: Notes about the usage of this command:**<br>
+
+* Edits the student at the specified `INDEX` with the specified updated particular to be updated.
 * The index refers to the index number shown in the displayed student list.
 * The index **must be a positive integer** 1, 2, 3, …
 * User will then be prompted for a confirmation input **`(yes/no)`**.
+   * If yes: edits and updates the particulars of the specified student from the student list.
+   * If no: the specified student's particulars will not be edited.
+   
+</div>
+
+**Format:**
+```
+editstu INDEX n/UPDATED_NAME
+```
+
+**Expected result:**
+```
+Edit INDEX? (yes/no)
+```
+
+:tip: After receiving the confirmation prompt, enter `yes` to confirm the editing of student's particulars at the specified `INDEX`, or else, reject the command by entering `no`.
+
+**Format:**
+```
+yes
+```
+
+**Expected result:**
+```
+Edited student: PARTICULARS_OF_STUDENT_AT_SPECIFIED_INDEX
+```
+   ![ScreenshotEditstu](images/ScreenshotEditstu.png)
 
 Examples:
-* `liststu` followed by `deletestu 2` followed by `yes` deletes the 2nd student in the student list.
-* `findstu Betsy` followed by `deletestu 1` followed by `y` deletes the 1st student in the results of the `findstu` command.
+* `editstu 2 n/John Cena` followed by `yes` edits the 2nd student in the student list with an updated name `John Cena`.
+* `editstu 3 t/Joker` followed by `y` edits the 3rd student in the student list with an updated tag `Joker`.
+
+{ end of `features#student#editstu` written by: Marcus Tan Wei }
+
+{ start of `features#student#clearstu` written by: Marcus Tan Wei }
 
 #### 5.3.6. Clearing the student list : `clearstu`
 
-The user will be prompted to confirm their decision here.
+Clears and removes all students from the student list.
+
+<div markdown="block" class="alert alert-info">
 
-If yes: removes all students from the student list.
+**:information_source: Notes about the usage of this command:**<br>
+* The user will be prompted to confirm their decision here.
+   * If yes: removes all students from the student list.
```

Comment:
Perhaps could have more consistent usage of active vs passive voice?

Reply:
Use of "you" here?